source/accounts/vsc_user_groups.rst

```diff
@@ -124,5 +125,43 @@ Working with file and directory permissions
 
       ::
 
-         [user@foo]# find /path/to/directory -type d -exec chmod g+s '{}' \\;
+         find /path/to/directory -type d -exec chmod g+s '{}' \\;
+
+
+.. _sharing_via_vsc_groups:
+
+Sharing files and folders with other VSC users
+----------------------------------------------
+
+VSC users can share files/folders with other VSC users only via :ref:`VSC groups <vsc_groups>`.
+Imagine a VSC user who wants to share his post-processed data stored below the
+``$VSC_DATA/results/post-processed`` folder with another VSC user(s).
+Then, these are the steps to take:
+
+- Create a :ref:`new VSC group <create groups>` (if existing groups do not suit your purpose).
+  Let's call this new group ``lp_shared_data``.
+- Invite your VSC collaborators to join the group ``lp_shared_data``.
+- Change the group name of the folders in the hierarchy one by one to ``lp_shared_data``:
+
+  ::
+
+     chgrp lp_shared_data $VSC_DATA
+     chgrp lp_shared_data $VSC_DATA/results
+     # change group name recursively below the top hierarchy you want to share
+     chgrp -R lp_shared_data $VSC_DATA/results/post-processed
+
+- Make sure that the shared directory has read and execute access to all folders underneat,
```

Comment:
This phrase is not very clear. I guess you mean that, in this example, `$VSC_DATA` and `$VSC_DATA/results` need to be readable and executable by the group `lp_shared_data`?